testproject/ProjectSettings/BurstAotSettings_StandaloneWindows.json

```diff
@@ -1,6 +1,6 @@
 {
   "MonoBehaviour": {
-    "Version": 3,
+    "Version": 4,
```

Comment:
we probably don't need changes in this file :P

Reply:
Oops... I usually notice these json files and unstage them. Was in a hurry last night and missed it.